.github/workflows/publish.yaml

```diff
@@ -83,18 +98,60 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and push to GAR and Docker Hub
+      - name: Build and push to GAR
+        if: ${{ github.event_name == 'push' }}
         uses: docker/build-push-action@v6
         with:
           context: .
           file: RemoteSettings.Dockerfile
+          sbom: true
           target: production  # See multi-stage build
           push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          tags: ${{ steps.metagar.outputs.tags }}
+          labels: ${{ steps.metagar.outputs.labels }}
           platforms: linux/amd64,linux/arm64
           cache-from: type=gha # Load cache from GitHub Actions
           cache-to: type=gha,mode=max # Save cache to GitHub Actions
+      - name: Set up crane # copied from https://github.com/imjasonh/setup-crane/blob/main/action.yml
```

Comment:
Why do we need crane? ~Retagging images should be [possible with the normal Docker CLI](https://docs.docker.com/reference/cli/docker/buildx/imagetools/create/), which is already installed on the runner image.